src/OrchardCore.Modules/OrchardCore.Settings/Drivers/DefaultSiteSettingsDisplayDriver.cs

```diff
@@ -104,5 +109,8 @@ private static void PopulateProperties(ISite site, SiteSettingsViewModel model)
             model.AppendVersion = site.AppendVersion;
             model.CacheMode = site.CacheMode;
         }
+
+        private static bool IsGeneralSettings(BuildEditorContext context)
```

Comment:
@hishamco 
Maybe rename this to `IsGeneralGroup`. Ping me once done and I'll merge it